<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SSHTunneller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-21</y>
      <width>381</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="homepage">
     <widget class="QTabWidget" name="tabContainer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>381</width>
        <height>261</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>231</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>369</width>
           <height>229</height>
          </rect>
         </property>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>231</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Streams</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QPushButton" name="icon">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight:600</string>
      </property>
      <property name="text">
       <string>SSHTunneller</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#connectButton{
border-radius: 50%;
border: 10px solid blue;
}</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>26</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>480</y>
        <width>110</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="payloadInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>336</width>
        <height>74</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="payloadLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Payload</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Host:Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="usernameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
